src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [people, setPeople] = useState<Person[] | null>(null);
+
+  useEffect(() => {
+    getPeople()
+      .then(res => {
+        setPeople(res);
+      })
+      .catch(() => {
+        //
```

Comment:
It's better to catch errors here. The empty array doesn't show problems with loading data

Reply:
I trully agree with you... BUT if I throw an error here 1 test fails because it says I should not throw errors.
To be honest I was just tired to look for that test and try to understand where should  I operate that error to keep it green.